plugins/webp-uploads/tests/test-load.php

```diff
@@ -850,20 +850,6 @@ public function test_it_should_create_webp_for_some_sizes_which_are_smaller_in_w
 		$this->assertSame( $expected_tag, $updated_tag );
 	}
 
-	/**
-	 * Tests that the fallback script is not added when a post with no updated images is rendered.
-	 */
-	public function test_it_should_not_add_fallback_script_if_content_has_no_updated_images(): void {
-		remove_all_actions( 'wp_footer' );
-
-		apply_filters( 'the_content', '<p>no image</p>' );
-
-		$this->assertFalse( has_action( 'wp_footer', 'webp_uploads_webp_fallback' ) );
-
-		$footer = get_echo( 'wp_footer' );
-		$this->assertStringNotContainsString( 'webp;base64,UklGR', $footer );
-	}
-
```

Comment:
This should have been removed as part of https://github.com/WordPress/performance/pull/1269